### DISCUSSION TITLE
Welcome to Open-Source Discussions!

### DISCUSSION THREAD

**Discussion by vimal-verma** [IIITians-Network/Open-Source#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by oscarscfrederiksen** [comment]
Hey! Thanks for this awesome space! I just came here to hopefully get some contributions to my repo and contribute to others. I recently started compiling a repo called "Useful-Packages" which aims to be a massive composition of everyone's favourite packages, APIs and the like for all languages, systems and platforms. It would be awesome if you could check it out and contribute; the link is: https://github.com/oscarscfrederiksen/Useful-Packages.

**Reply by vimal-verma** [reply]
Hello @oscarscfrederiksen  ,Thanks for Sharing That Awesome Repository with us, Nowadays there are many package on Package manager, And it really hard to chose which one is better, I hope that Repository helps beginners.
